Infrastructure/Storage/Repositories/CustomerRepository.cs

```diff
@@ -127,6 +128,38 @@ public static void UpdateCustomer(Customer customer)
             }
         }
 
+        internal static void DisableCustomer(int customerId)
+        {
+            using (SQLiteConnection connection = DbConnectionHelper.GetDatabaseConnection(DbConnectionHelper.SparePartConfig) as SQLiteConnection)
+            {
+                connection.Open();
+                using (SQLiteCommand cmd = new SQLiteCommand("", connection))
+                {
+                    var customer = FindCustomer(customerId);
+                    customer.Enabled = false;
+                    UpdateCustomer(customer, cmd);
+                }
+
+                connection.Close();
+            }
+        }
+
+        internal static void EnableCustomer(int customerID)
+        {
+            using (SQLiteConnection connection = DbConnectionHelper.GetDatabaseConnection(DbConnectionHelper.SparePartConfig) as SQLiteConnection)
+            {
+                connection.Open();
+                using (SQLiteCommand cmd = new SQLiteCommand("", connection))
```

Comment:
There is no need for two steps here. YOu can use just one SQL command to directly update the field's value

Reply:
edited